src/lib/supports_pixelated_image.js

```diff
@@ -15,7 +15,8 @@ function supportsPixelatedImage() {
     if(_supportsPixelated !== null) { // only run the feature detection once
         return _supportsPixelated;
     }
-    if(Lib.isIE()) {
+    if(Lib.isIE() || Lib.isSafari()) {
```

Comment:
Per the discussion in #6604 seems like this is a problem with all webkit browsers. Unfortunately Chrome [say it's webkit](https://security.stackexchange.com/questions/126407/why-does-chrome-send-four-browsers-in-the-user-agent-header) in its `userAgent` so perhaps the best we can do is also include iOS?

```suggestion
    if(Lib.isIE() || Lib.isSafari() || lib.isIOS()) {
```

Reply:
Yes for now it sounds more reasonable to just exclude Safari/IOS. I was searching for a way to clearly identify the actual rendering engine in use, but feel like it won't be safe or it would require more time to investigate further.